javascripts/discourse/services/homepage-filter.js

```diff
@@ -14,6 +14,7 @@ export default class HomepageFilter extends Service {
   @tracked loading = false;
   @tracked hasMoreResults = false;
   @tracked currentPage = 1;
+  @tracked locale = localStorage.getItem("DiscoverLocale") || "locale-en";
```

Comment:
I'm not a huge fan of storing the language choice in local storage. We don't have a lot of sites in non-English locales, so once a user switches locale, they'll see very few results in each tag-filtered list. 

Could we have this locale dropdown just filter the list to that locale's forums? That way the other filters remain as they currently are. (We'd be lying a bit by "All" being limited to English, but I think we should still keep the default view to English too.)